model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/InfinispanUserSessionProvider.java

```diff
@@ -370,13 +370,13 @@ protected Stream<UserSessionModel> getUserSessionsStream(RealmModel realm, UserS
                 String[] idpAliasSessionId = predicate.getBrokerUserId().split("\\.");
```

Comment:
Splitting the `brokerUserId` at all dots is fragile, and was probably broken before when a user contained a dot (which is quite likely) or an IDP alias contained a dot (possible, still less likely). 

Once the issuer is being used, those issuers will be URLs which will contain dots a lot of the time. So I'd suggest to avoid concatenating those values, and keep them separate throughout Keycloak.

Reply:
The alias, AFAIK, does not accept certain special characters. See org.keycloak.utils.ReservedCharValidator#RESERVED_CHARS_PATTERN. 

In any case, I would like to not fix this now as it is a bit beyond the scope of my changes.